product/dialogs/miq_dialogs/miq_provision_redhat_dialogs_template.yaml

```diff
@@ -418,8 +418,8 @@
               :dvs: true
               :vlans: true
             :method: :allowed_vlans
-          :description: vLan
-          :required: true
+          :description: Virtual Nic Profile
```

Comment:
Shouldn't the 'Virtual Nic Profile' be used as a header as a dependency of the provider highest supported version ?